poly-commit/src/utils.rs

```diff
@@ -0,0 +1,141 @@
+use ark_ff::Field;
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use ark_std::vec::Vec;
+
+#[cfg(feature = "parallel")]
+use rayon::{
+    iter::{IntoParallelRefIterator, ParallelIterator},
+    prelude::IndexedParallelIterator,
+};
+
+/// Takes as input a struct, and converts them to a series of bytes. All traits
+/// that implement `CanonicalSerialize` can be automatically converted to bytes
+/// in this manner.
+/// From jellyfish lib
+#[macro_export]
+macro_rules! to_bytes {
+    ($x:expr) => {{
+        let mut buf = ark_std::vec![];
+        ark_serialize::CanonicalSerialize::serialize_compressed($x, &mut buf).map(|_| buf)
+    }};
+}
```

Comment:
`ark-std` now contains a `serialize_to_vec` macro that you might want to use instead of this.

Reply:
Done!